msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -193,11 +257,87 @@ You can also deploy the samples to Azure using Teams Toolkit. To deploy, follow
 
 ::: zone-end
 
+::: zone pivot="qs-python"
+
+## Build and run the sample app
+
+Before get started, ensure that you've installed the [python v3.8.10](https://www.python.org/) and [poetry](https://python-poetry.org/docs/#installing-with-pipx) extensions.
+
+1. Go to the [sample](https://github.com/microsoft/teams-ai/tree/main/python/samples).
+
+1. Clone the repository to test the sample app.
+
+   ```
+   git clone https://github.com/microsoft/teams-ai.git
+   ```
+
+1. Go to the **python** folder.
+
+   ```
+   cd teams-ai/python
+   ```
+
+1. Go to the folder where you cloned the repository and select **04.ai.c.actionMapping.lightBot**. All the files for the Light bot sample are listed under the **EXPLORER** section in Visual Studio Code.
+
+1. Under **EXPLORER**, duplicate the **sample.env** file and update the duplicate file to **.env**.
+
+      # [OpenAI key](#tab/OpenAI-key2)
+
+      Go to `env` folder and update the following code in `./env/.env.local.user` file:
+
+      ```text
+      SECRET_OPENAI_KEY=<your OpenAI key>
+
+      ```
+
+      # [Azure OpenAI](#tab/Azure-OpenAI2)
+
+      Go to `env` folder and update the following code in `./env/.env.local.user` file:
+
+      ```text
+      SECRET_AZURE_OPENAI_KEY=<your Azure OpenAI key>
+      SECRET_AZURE_OPENAI_ENDPOINT=<your Azure OpenAI Endpoint>
+
+      ```
+
+      ---
+
+1. To install the following dependencies, go to **View** > **Terminal** and run the following commands:
```

Comment:
These commands don't need to be run. All you need is run is:

`poetry install` and
`poetry build`

Reply:
As discussed, we'd not incorporate this comment as dependencies are needed to get the output.